polygeist/tools/cgeist/Test/Verification/sycl/single_task.cpp

```diff
@@ -0,0 +1,27 @@
+// RUN: sycl-clang.py %s -S -emit-llvm -o %t.ll
+// Test that the LLVMIR generated is verifiable.
+// RUN: opt -verify -disable-output < %t.ll
+// RUN: cat %t.ll | FileCheck %s
```

Comment:
[SUGGESTION]: We should also add a step to verify that the LLVM IR generated is translatable to SPIRV without errors.